index.html

```diff
@@ -1,42 +1,157 @@
-    </section>
-    <section>
-            <div class="conn-page">
-                <div><p class="connect">LET'S CONNECT</p></div>
-                <div><img class="img4" src="images/el4.png" ></div>
-            </div>
+
+            <nav class="toolbar">
+                <div class="logo"><a href="#">My Logo</a></div>
+                <div class="menu-icon"><img src="images/menu.png" alt="follow me on angellist" width="24" height="24" ></div>
+            </nav>
+         </header>
+             
+             <section class="first-page">
+          <div class="div-header">
+
+           <h1 class="header">I'm Kevin Glad to see you!</h1>  
+         <p class="about-par">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+               
+                    <a class="connect-page">LET'S CONNECT</a>
+                
+                <ul class="list">
+                    <li class="list-item"><img src="images/tweeter.png" alt="follow me on twitter" width="20" height="20" ></li>
+                    <li class="list-item"><img src="images/linkedin.png" alt="follow me on linkedin" width="20" height="20" ></li>
+                    <li class="list-item"><img src="images/medium.png" alt="follow me on medium" width="20" height="20" ></li>
+                    <li class="list-item"><img src="images/github.png" alt="follow me on github" width="20" height="20" ></li>
+                    <li class="list-item"><img src="images/angelist.png" alt="follow me on angellist" width="20" height="20" ></li>
+                </ul>
+                </div>
+        </section>  
+
+            <section class="work-section">
+               <div class="card">
+            <div class="section-img"><img src="images/snapshoot1.png"></div>
+            <div class="card-info">
+            <h2 class="work-header">Tonic</h2>
+            <ul class="descr-list">
+                <li class="descr-list-item1">CANOPY</li>
+                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png"> End Dev</li>
+                <li class="descr-list-item"><img class="counter-img"src="images/Counter.png">2015</li>
+            </ul>
+            <p class="lang-descr">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="lang-list">
+                <li class="lang-list-descr">html</li>
+                <li  class="lang-list-descr">css</li>
+                <li  class="lang-list-descr">javaScript</li>
+            </ul>
+            <button class="btn-project">See Project</button>
+        </div>
```

Comment:
- [ ] While you did a good job implementing the required changes :+1: , please don't forget to add a margin between the headline section and the work section. Because as project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0)

**Your design:**

![Screenshot from 2023-01-12 19-59-19](https://user-images.githubusercontent.com/105117832/212212983-b0205245-a7b8-416f-a76e-154014c549d0.png)

**Figma design:**

![Screenshot from 2023-01-12 20-02-13](https://user-images.githubusercontent.com/105117832/212213130-82e55f23-97bd-4e30-9394-cee83b543876.png)

Reply:
@IngDavidTM can I use autoprefixer in the css files?